projects/skia/build.sh

```diff
@@ -129,10 +129,12 @@ cp ./canvas_seed_corpus.zip $OUT/api_null_canvas_seed_corpus.zip
 
 # Remove unnecessary dependencies that aren't on runner containers.
 # Libraries found through trial and error (ldd command also helpful).
-patchelf --remove-needed libGLU.so.1 out/GPU/api_null_gl_canvas
-patchelf --remove-needed libGL.so.1 out/GPU/api_null_gl_canvas
-patchelf --remove-needed libX11.so.6 out/GPU/api_null_gl_canvas
-cp out/GPU/api_null_gl_canvas $OUT/api_null_gl_canvas
+patchelf --remove-needed libGLU.so.1 out/GPU/api_mock_gpu_canvas
+patchelf --remove-needed libGL.so.1 out/GPU/api_mock_gpu_canvas
+patchelf --remove-needed libX11.so.6 out/GPU/api_mock_gpu_canvas
+cp out/GPU/api_mock_gpu_canvas $OUT/api_mock_gpu_canvas
+cp ./api_mock_gpu_canvas.options $OUT/mock_gpu_canvas.options
+cp ./canvas_seed_corpus.zip $OUT/api_mock_gpu_canvas_seed_corpus.zip
```

Comment:
I think you need to add a command to download this in the Dockerfile.

Reply:
canvas_seed_corpus.zip is already there (and used by other canvas fuzzers)

Reply:
Ah right, my bad.